File: platforms/ios/Dezel/Dezel/View/TextViewObserver.swift
/**
 * @protocol TextViewObserver
 * @since 0.7.0
 */
public protocol TextViewObserver: class {

	/**
	 * @method onPressLink
	 * @since 0.7.0
	 */
	func didPressLink(textView: TextView, url: String)

}

File: platforms/android/Dezel/dezel/src/main/java/ca/logaritm/dezel/view/TextViewObserver.kt
package ca.logaritm.dezel.view

/**
 * @interface TextViewObserver
 * @since 0.7.0
 */
public interface TextViewObserver {

	/**
	 * @method onPressLink
	 * @since 0.7.0
	 */
	fun onPressLink(textView: TextView, url: String)

}
